scripts/build-colors.js

```diff
@@ -228,6 +228,11 @@ extension UIColor {
         let named = inHighContrast ? "\\(name.rawValue)HighContrast" : name.rawValue
         return UIColor(named: named, in: .core, compatibleWith: nil)!
     }
+
+    @objc public static func named(_ name: String) -> UIColor? {
+        guard let color = InstColorName(rawValue: name) else { return nil }
+        return named(color)
+    }
```

Comment:
good job adding the helper to the script so it doesn't get lost the next time it is run.